src/ITensors.jl

```diff
@@ -149,6 +156,7 @@ include("nullspace.jl")
 #####################################
 # Ops to ITensor conversions
 #
+include("Ops/permutations.jl")
```

Comment:
Same as above, I don't think `Ops` is the right location for this file.

Reply:
Same as above